asdf/tests/test_yaml.py

```diff
@@ -97,14 +109,30 @@ def test_python_tuple(tmpdir):
         "val": (1, 2, 3)
     }
 
-    def check_asdf(asdf):
-        assert isinstance(asdf.tree['val'], list)
+    run_tuple_test(tree, tmpdir)
+                                  
+                                  
+def test_named_tuple_collections(tmpdir):
```

Comment:
Can you please add a test showing that recursive serialization of `namedtuple` members is handled correctly? For example, you could create a `namedtuple` where one of the members is an `np.ndarray`.